components/stepper/linear-flow.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Linear Flow
+page_title: Linear Flow
+description: Linear Flow of the Stepper for Blazor.
+slug: stepper-linear-flow
+tags: telerik,blazor,stepper,linear,flow
+published: True
+position: 20
+---
+
+# Linear Flow
+
+The `Linear` flow property of the Stepper component allows you to configure it, so that completion of the previous step is required before proceeding to the next step. The user will be able to go one step at a time (back or forth).
```

Comment:
Consider simplifying the language and sentence a little.

Reply:
Done